src/main/webapp/app/entities/course/course-exercise-card.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="question card">
+    <div class="question-options card-header question-card-header">
+        <h5 [routerLink]="exerciseRouterLink"><span [jhiTranslate]="headingJhiTranslate"></span></h5>
+        <button class="btn btn-light question-collapse" (click)="isCollapsed = !isCollapsed"
+                [attr.aria-expanded]="!isCollapsed" [attr.aria-controls]="'collapseQuestion'">
+            <span class="fa fa-2x" [ngClass]="isCollapsed ? 'fa-angle-right' : 'fa-angle-down'"></span>
```

Comment:
Please, merge `develop` in your branch and use the `<fa-icon>` component :)

You can see more details here, with some examples of replacements: https://github.com/ls1intum/ArTEMiS/pull/157